src/commands/classified/external.rs

```diff
@@ -56,8 +157,7 @@ pub(crate) async fn run_external_command(
 ) -> Result<Option<InputStream>, ShellError> {
     trace!(target: "nu::run::external", "-> {}", command.name);
 
-    let has_it_arg = command.args.iter().any(|arg| arg.contains("$it"));
-    if has_it_arg {
+    if command.has("$it") {
```

Comment:
`has` is for named arguments not positional ones, so I don't believe this does what you want.

Reply:
Same, should we name it differently? `has` returns whether it found an `$it` in it's arguments or not.

Reply:
I think if "it's found an $it in its named arguments". But $it would be a positional argument

Reply:
What should we call the function?

Reply:
.has() is fine, probably, I just got confused because it works differently than other .has() functions, like the one in CallInfo